targets/TARGET_STM/README.md

```diff
@@ -1,14 +1,57 @@
-## README for mbed-os STM32 targets
+# README for mbed-os STM32 targets
 
-### MBED Wiki pages
+## ST TOOLS
 
-https://os.mbed.com/teams/ST/wiki/
+### USB drivers
+
+Mandatory: get the latest USB driver in order to make available all the USB interfaces provided by the ST-LINK:
+- ST Debug
+- Virtual COM port
+- ST Bridge interfaces
+
+https://www.st.com/en/development-tools/stsw-link009.html
```

Comment:
What's the need to use this specific stsw-link009? 
I don't recall installing any of that and boards are detected just fine.
It would help to elaborate a bit on this.

Reply:
I used the [stsw-link007](https://www.st.com/en/development-tools/stsw-link007.html) utility on W10 to update the fw interface and didn't have to install USB drivers, but maybe these are installed automatically as part of the 007 package?
Anyway, I think it would be good to elaborate a bit on what's that package and intended for. 
Can you add a few lines?

Reply:
Done few lines later :-)